From: SeongJae Park <sj@kernel.org>
Subject: Docs/mm/damon/design: fix a typo on 'intervals auto-tuning' section
Date: Wed, 5 Mar 2025 10:27:44 -0800

Commit ("Docs/mm/damon/design: document for intervals auto-tuning") made a
typo on a cross reference link in monitoring intervals auto-tuning
section.  It causes document build warning.  Fix it.

Link: https://lkml.kernel.org/r/20250305182744.56125-1-sj@kernel.org
Signed-off-by: SeongJae Park <sj@kernel.org>
Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Closes: https://lore.kernel.org/20250305203312.6f30e9c2@canb.auug.org.au
Cc: Jonathan Corbet <corbet@lwn.net>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 Documentation/mm/damon/design.rst |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/Documentation/mm/damon/design.rst~docs-mm-damon-design-document-for-intervals-auto-tuning-fix
+++ a/Documentation/mm/damon/design.rst
@@ -340,7 +340,7 @@ DAMON-observed access events to the theo
 (``aggrs``).
 
 The DAMON-observed access events are calculated in byte granularity based on
-DAMON :ref:`region assumption <damon_design_region_based_sample>`.  For
+DAMON :ref:`region assumption <damon_design_region_based_sampling>`.  For
 example, if a region of size ``X`` bytes of ``Y`` ``nr_accesses`` is found, it
 means ``X * Y`` access events are observed by DAMON.  Theoretical maximum
 access events for the region is calculated in same way, but replacing ``Y``
_
